metadata-ingestion/docs/sources/azure-data-factory/azure-data-factory_pre.md

```diff
@@ -0,0 +1,209 @@
+| Lineage (Column-level) | 🔜     | Coming soon for Copy Activity mappings             |
+| Pipeline-to-Pipeline   | ✅     | ExecutePipeline activities create lineage          |
+| Data Flow Scripts      | ✅     | Stored as transformation logic                     |
+| Execution History      | ✅     | Enabled by default via `include_execution_history` |
+| Stateful Ingestion     | ✅     | Stale entity removal                               |
+
+## Prerequisites
+
+### Authentication
+
+The connector supports multiple authentication methods:
+
+| Method                     | Best For                                         | Configuration                                       |
+| -------------------------- | ------------------------------------------------ | --------------------------------------------------- |
+| **Service Principal**      | Production environments                          | `authentication_method: service_principal`          |
+| **Managed Identity**       | Azure-hosted deployments (VMs, AKS, App Service) | `authentication_method: managed_identity`           |
+| **Azure CLI**              | Local development                                | `authentication_method: cli` (run `az login` first) |
+| **DefaultAzureCredential** | Flexible environments                            | `authentication_method: default`                    |
+
+For service principal setup, see [Register an application with Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app).
+
+### Required Permissions
+
+The connector only performs **read operations**. Grant one of the following:
+
+**Option 1: Built-in Reader Role** (recommended)
+
+Assign the [Reader](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#reader) role at subscription, resource group, or Data Factory level.
+
+**Option 2: Custom Role with Minimal Permissions**
+
+Download [`datahub-adf-reader-role.json`](./datahub-adf-reader-role.json), update the `{subscription-id}`, then:
+
+```bash
+# Create custom role
+az role definition create --role-definition datahub-adf-reader-role.json
+
+# Assign to service principal
+az role assignment create \
+  --assignee <service-principal-id> \
+  --role "DataHub ADF Reader" \
+  --scope /subscriptions/{subscription-id}
+```
+
+For detailed instructions, see [Azure custom roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles).
+
+## Lineage
+
+The connector extracts lineage from:
+
+1. **Copy Activities**: Maps input/output datasets to DataHub datasets
+2. **Data Flow Activities**: Extracts sources and sinks, plus transformation scripts
+3. **Lookup Activities**: Maps lookup datasets as inputs
+4. **ExecutePipeline Activities**: Creates pipeline-to-pipeline lineage
+
+:::tip Column-Level Lineage Coming Soon
+Column-level lineage support is planned for Copy Activity `translator.columnMappings`.
+:::
```

Comment:
I'm reluctant to add notes like this becasue we are inconsistent with following up/resolving the notes in our docs in the long-run. Not a merge blocker, but please be mindful of how much docs debt we accumulate vs. our ability/practice to address it/clean it up later